examples/IBFE/explicit/ex9/example.cpp

```diff
@@ -283,7 +283,7 @@ main(int argc, char* argv[])
         EquationSystems* bndry_equation_systems = ib_method_ops->getFEDataManager()->getEquationSystems();
 
         // Setup solid systems.
-        std::unique_ptr<EquationSystems> solid_equation_systems(new EquationSystems(solid_mesh));
+        std::unique_ptr<EquationSystems> solid_equation_systems = std::make_unique<EquationSystems>(solid_mesh);
```

Comment:
```suggestion
        auto solid_equation_systems = std::make_unique<EquationSystems>(solid_mesh);
```
We can't do this with ternary ifs but we could do it it here.

Reply:
Ah my regex did not find those.